src/main/java/com/exactpro/th2/check1/event/CheckSequenceUtils.java

```diff
@@ -142,6 +143,9 @@ private static String getKeyFields(String name, ValueFilter valueFilter) {
 
     private static String getKeyFields(String name, SimpleFilter valueFilter) {
         if (valueFilter.getKey()) {
+            if (valueFilter.hasSimpleList() && (valueFilter.getOperation() == FilterOperation.IN || valueFilter.getOperation() == FilterOperation.NOT_IN)) {
+                return ", " + name + '=' + valueFilter.getSimpleList();
```

Comment:
We cannot use the `simpleList` as is. It is a protobuf object and the `toString` method won't create the representation we need. We should join the values into a string by ourselves.
Also, I think we should use `in` instead of `=` here

Reply:
Is it correct view: , {name} NOT_IN ["simpleValue1", simpleValue2]?